modules/test/playwright/tests/commerce/commerceProducts.spec.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+// @ts-ignore
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {commercePagesTest} from '../../fixtures/commercePagesTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {getRandomInt} from '../../utils/util';
+
+export const test = mergeTests(apiHelpersTest, commercePagesTest, loginTest);
+
+test('LPD-5780 modal title and product name appear properly in product menu', async ({
+	apiHelpers,
+	commerceProductAdminPage,
+}) => {
+	const catalog = await apiHelpers.headlessCommerceAdminCatalog.postCatalog({
+		name: 'Product Catalog',
+	});
+
+	const product1 = await apiHelpers.headlessCommerceAdminCatalog.postProduct({
+		catalogId: catalog.id,
+		name: {
+			en_US: '"Product' + getRandomInt(),
+		},
```

Comment:
Do we need to pass this name? (I think it's the same as autogenerated by the method itself if it doesn't receive it in input) - like the product2 that doesn't specify the name

Reply:
We do need to pass this name because COMMERCE-13162 is specifically related to double-escaped characters appearing in certain areas of the UI. With a completely autogenerated name we have no guarantee that there will be a character in the name that would normally be escaped as far as I know.

Reply:
But also this name I think doesn't contains "strange chars" --- it should always generate something like Product12345 or Product35267

Reply:
It does contain an escapable char. It's a bit hard to see but note the quotation mark inside the single quote mark:

'**"**Product' + getRandomInt(),

So it generates "Product12345 or "Product32393.

Reply:
My fault, didn't see the " at the beginning of the name. It's fine